### DISCUSSION TITLE
No server is currently available to service your request

### DISCUSSION THREAD

**Discussion by ahm3dkira** [community/community#32126] [Other Feature Feedback, Questions, & Ideas]
No server is currently available to service your request.
This happens to me multiable times a week for quite some time.
stays like that for 2-5 min then goes up again.
only to github.com website is affected domain other github serves like githubstatus.com, support.github.com is up as normal
any suggestions?

https://downdetector.com/status/github/

Note that my internet is not down; this happens whether my internet is fast or slow. (fast ~24 Mbps; slow ~1Mbps)

![Screenshot from 2022-09-06 13-47-10](https://user-images.githubusercontent.com/50806393/188628147-e99b86b1-1f8d-425d-8e28-5fc269750d8d.png)
![Screenshot from 2022-09-06 13-47-46](https://user-images.githubusercontent.com/50806393/188628160-1aed9049-ffb3-4d39-a454-f5309dbfadcf.png)
![Screenshot from 2022-09-06 13-48-20
](https://user-images.githubusercontent.com/50806393/188628176-e027170f-232f-4ecc-b34b-dcd69c0dc1c1.png)

**Comment by phoeniixhawk** [comment]
I'm seeing this too, about once a week, and usually coming from LFS downloads or submodule checkouts.

**Comment by kun2001github** [comment]
Hi, I also encountered the same problem, can you solve it?

**Comment by surabh-data** [comment]
I am seeing this today. Did you come to know the reason behind it

**Comment by shivam-dev-stack** [comment]
i am seeing this too, mobile version is working good and web is not working fine

**Reply by JustAlex-WebDev** [reply]
i am seeking it too.

**Comment by david-engelmann** [comment]
https://www.githubstatus.com/ -> There is an outage right now

**Comment by david-engelmann** [comment]
https://www.githubstatus.com/ -> There is an outage right now

**Comment by chrkaatz** [comment]
I see the same - cannot assign people to tickets or update the status of a ticket on the Board

**Comment by zinmyoswe** [comment]
i am also facing the problem after i restart the PC.

**Comment by tophersymps** [comment]
facing this just trying to login.. but did get logged in now

**Comment by vinodhj** [comment]
GitHub is also experiencing an outage.

**Comment by veeresh-machani** [comment]
https://www.githubstatus.com/ 

Outage issue

**Comment by JitBay** [comment]
https://www.githubstatus.com/ -> There is an outage right now

**Comment by AvdylDemukajj** [comment]
Probably you hit github api rate limits or github is down

**Comment by morganpatterson46795-cmyk** [comment]
I’ve seen this error before, and it can be frustrating when you’re trying to get a request processed. Checking the service status and retrying after a short wait may help. For related restaurant information,https://mylongjohnsilversexperience.xyz can also be useful. Hopefully the issue gets resolved soon.
